src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  const onLeftClick = (event) => {
+    if (event.button === 0) {
+      resolve('First promise was resolved');
+    }
+  };
+
+  document.addEventListener('click', onLeftClick);
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('click', (event) => {
+    if (event.button === 0) {
+      resolve('Second promise was resolved');
+    }
+  });
+
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('contextmenu', (event) => {
+    event.preventDefault();
+    resolve('Second promise was resolved');
```

Comment:
The second promise resolves on both left and right clicks. If the intention is to resolve only on a specific type of click, you should revise the logic to differentiate between left and right clicks.